### PR TITLE
ChatList v1 design: header + avatar icon

### PR DESCRIPTION
**Pull request by tgfassina**
# Description
- https://www.pivotaltracker.com/story/show/182056283
- partial implementation. includes the header + avatar icon

image

<sub> Mention [stepsize] in a comment if you'd like to report some technical debt. See examples [here](https://app.stepsize.com/api/demo-pr-redirect).</sub>

### CODE REVIEW

File: App.tsx [hunk 1]
```diff
@@ -1,18 +1,39 @@
...
 import ChatScreen from "./src/screens/Chat";
+
+// import { createBottomTabNavigator } from "@react-navigation/bottom-tabs";
 // import NearbyScreen from "./src/screens/Nearby";
 // import ProfileScreen from "./src/screens/Profile";
 
 // const Tab = createBottomTabNavigator();
 
 export default function App() {
+  const [fontsLoaded] = useFonts({
```
Comment (matheuspb) [inline App.tsx:25]: <3

**Review by matheuspb — APPROVED** [review]

File: App.tsx (hunk 1 above)
Comment (tgfassina) [reply App.tsx:25]: hahah
Do you know if there's a way to import `useFonts` standalone?
The fact that it gets imported as part of a random font called poppins triggers my OCD haha

**Review by pererinha — APPROVED** [review]
lgtm!

File: App.tsx (hunk 1 above)
Comment (matheuspb) [reply App.tsx:25]: lol, but not that I know

**Pull request merged**